seed/lib/mappings/data/pm-mapping.json

```diff
@@ -170,12 +188,21 @@
     "type": "float",
     "schema": ""
   },
+  {
+    "display_name": "National Median Site Energy Use",
+    "to_field": "national_median_site_energy_use",
+    "to_table_name": "PropertyState",
+    "from_field": "National Median Site Energy Use",
+    "units": "kBtu",
+    "type": "float",
+    "schema": ""
+  },
   {
     "display_name": "National Median Site EUI",
     "to_field": "national_median_site_eui",
     "to_table_name": "PropertyState",
     "from_field": "medianSiteIntensity",
-    "units": "kBtu/ft2",
+    "units": "kBtu/ft**2",
     "type": "float",
     "schema": ""
```

Comment:
Fields like this are present on the "Full list of Portfolio Manager custom reporting metrics" but not on the ESPM export. Since it is existing I chose to leave it in.

Reply:
Actually, I think we need to keep these as ft**2, which we check here:
https://github.com/SEED-platform/seed/blob/2898863714434ce928231a4f69563b2c4f877100/seed/utils/organizations.py#L30